ios-swift/App/Realm.plist

```diff
@@ -2,6 +2,8 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>baseUrl</key>
```

Comment:
nit: should we make this consistent across the SDKs? The [spec](https://docs.google.com/document/d/1gmbjUjeSULW4aI3OV9ytBqU0sypQ78H2cpIKIHr1xWM/edit#) has some using `base-url` and others using just `url` (none currently say `baseUrl` though it's fine if we choose that)

Reply:
I'm actually a bit confused here - the spec lists both a `base-url` and `app-url` for iOS and Android but only a `url` for web 🤔

Reply:
Discussed in slack - will follow up with Andrew and make consistent across the board.

Reply:
yeah sorry that was my bad on the spec. 

I meant to make all of the clients export the same metadata, but I must've been distracted halfway